package.json

```diff
@@ -175,30 +174,34 @@
     "mongodb-memory-server": "^5.1.3",
     "nodemon": "^1.17.2",
     "npm-check": "^5.9.0",
+    "npm-run-all": "^4.1.5",
     "onchange": "~6.0.0",
     "prettier": "^1.15.3",
     "proxyquire": "~1.8.0",
     "regenerator-runtime": "^0.12.1",
     "supertest": "^4.0.2",
-    "ts-jest": "^24.0.2"
+    "ts-jest": "^24.0.2",
+    "ts-node": "^8.3.0",
+    "typescript": "^3.4.5"
   },
   "license": "MIT",
   "scripts": {
-    "start": "ts-node --files -r tsconfig-paths/register --project tsconfig.server.json app.ts",
+    "start": "ts-node --files -r tsconfig-paths/register --project tsconfig.server.json lib/app.ts",
     "dev": "concurrently --kill-others 'npm run dev:server' 'npm run dev:client' 'npm run watch'",
     "dev:server": "DEBUG=crowi:* NODE_ENV=development nodemon --watch ./lib/ --watch ./locales/ --ext 'ts,tsx,js,jsx,json' --exec npm run start",
     "dev:client": "NODE_ENV=development webpack --progress --profile --colors --display-error-details --watch --config tools/webpack.config.js",
     "format": "eslint --fix '**/*.{ts,tsx,js,jsx}'",
     "lint": "eslint '**/*.{ts,tsx,js,jsx}'",
     "test": "DEBUG= jest --maxWorkers=5",
     "coverage": "npm test -- --coverage",
-    "build": "npm run prod:client",
+    "build": "run-s build:*",
+    "build:server": "babel --no-babelrc --config-file ./.server.babelrc lib --out-dir dist --extensions '.ts'",
```

Comment:
Why using babel, I want to use `module-resolver` because `tsc` doesn't resolve `paths` to relative path.